elasticsearch/connection/http_urllib3.py

```diff
@@ -62,33 +76,37 @@ def __init__(self, host='localhost', port=9200, http_auth=None,
                 self.headers[k.lower()] = headers[k]
 
         self.headers.setdefault('content-type', 'application/json')
-        ca_certs = CA_CERTS if ca_certs is None else ca_certs
         pool_class = urllib3.HTTPConnectionPool
         kw = {}
         if use_ssl:
+            if not ca_certs and not ssl_context and verify_certs:
```

Comment:
yes, but that is in the code after this `if` so here it can still be empty since you removed the application of the default from line `65` - https://github.com/elastic/elasticsearch-py/pull/635/files#diff-e03df718a08bdad9d78d94c50dd7db90L65